test/UnitTest/Microsoft.Test.AspNet.OData/Query/FilterQueryOptionTest.cs

```diff
@@ -769,7 +770,6 @@ public void ApplyWithCast_ReturnsCorrectQueryable(string filter, int[] castModel
         [InlineData("cast(NoSuchProperty,Edm.String) eq null", typeof(ODataException))]
         [InlineData("cast(ProductId,Edm.NoSuchType) eq null", typeof(ODataException))]
         [InlineData("cast(ProductId,Edm.String) eq 123", typeof(ODataException))]
-        [InlineData("cast('123',Microsoft.Test.AspNet.OData.Common.Types.SimpleEnum) ne 'First'", typeof(ODataException))]
```

Comment:
Not sure why we are removing this test case. Is there a way to keep it and update the expected data of exceptionType properly instead?

Reply:
Not removed. Now, it's a valid case. Move it to the valid test case.
#Resolved.